ts/features/wallet/component/PaymentMethodCapabilities.tsx

```diff
@@ -79,6 +77,14 @@ const PaymentMethodCapabilities: React.FunctionComponent<Props> = props => {
       </View>
       <View spacer={true} />
       {capabilityItems.map(c => c)}
+      {capabilityItems.length !== 0 && (
```

Comment:
```suggestion
      {capabilityItems.length > 0 && (
```
It's the same but length can't be < 0